### ISSUE TITLE
Plex Homepage Filter glitch

### ISSUE THREAD

**Issue by jscottelblein** [causefx/Organizr#1210]
Using  v2.0.270, Master Branch in the organizrtools/organizr-v2 container

I'm noticing what might be a bug with the Plex homepage item. I have the "Recently Added" item at the top of the homepage, and have never used the filter until today. It was always showing all items as it should, but then I played around and selected each item form top to bottom in the filters list. Last one being Music. Now no matter which filter I choose, after the homepage does it's auto-refresh, it'll load up the music filter.

i.e. I select the Movies filter, it'll show only movies as it should, but after the auto refresh it goes back to the music filter, showing only the music items. This seems to be happening even through hard refreshes, and complete closing/reopening of Organizr, it starts with the Music filter on from that point on.

This happening with anyone else?

**Comment by causefx** [comment]
I can't replicate this on my main instance nor my dev instance.

**Comment by tronyx** [comment]
I am able to reproduce this in Firefox:

https://chrisyocum.com/Images/ShareX/2019-07-02_19-49-11.gif

Will try Chrome next.

**Comment by tronyx** [comment]
Happens in Chrome too (very long GIF is very long):

https://chrisyocum.com/Images/ShareX/2019-07-02_19-51-27.gif

**Comment by tronyx** [comment]
Scratch that. The two above GIFs prove two things:

1) I'm an idiot who cannot read.
2) I cannot reproduce your issue either.

**Comment by jscottelblein** [comment]
ok , I think I see what's going on, but it's still kinda weird. 

It seems to automatically reset the filter to "All" after each auto-refresh. And I guess when you do that it takes the latest X amount from every category and adds them to the list, even when for example, the last music was added 5 years ago, and 30 new movies have recently been added in the last say 10 days, it'll show the music right there near the front instead of all of those new movies first. I guess that's what is throwing me, because I haven't added any new music to Plex in years but they're showing up in the "Recently Added" right there in the front, before all the movies/shows I've recently added.

It seems like even when choosing the "All" filter, it should just be showing all the newest stuff solely based on date, no matter which category it comes from, then show the older stuff as you scroll right, from all the categories.

Also, I think it might be kinda nice to keep the selected filter through auto-refreshes so it doesn't keep "bouncing" back to "All" every time. Perhaps a session cookie to save/restore it's filter state or something.
